pkg/envoy/route/route_config.go

```diff
@@ -58,37 +58,33 @@ const (
 func BuildRouteConfiguration(inbound []*trafficpolicy.InboundTrafficPolicy, outbound []*trafficpolicy.OutboundTrafficPolicy, proxy *envoy.Proxy) []*xds_route.RouteConfiguration {
 	var routeConfiguration []*xds_route.RouteConfiguration
 
-	if len(inbound) > 0 {
-		inboundRouteConfig := NewRouteConfigurationStub(InboundRouteConfigName)
-		for _, in := range inbound {
-			virtualHost := buildVirtualHostStub(inboundVirtualHost, in.Name, in.Hostnames)
-			virtualHost.Routes = buildInboundRoutes(in.Rules)
-			inboundRouteConfig.VirtualHosts = append(inboundRouteConfig.VirtualHosts, virtualHost)
-		}
+	inboundRouteConfig := NewRouteConfigurationStub(InboundRouteConfigName)
```

Comment:
A few lines of comment regarding why an empty route-config could be sent down would be helpful.

Reply:
done